pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -33,6 +33,33 @@ class BasicSyncTestCase(unittest.TestCase):
     def setUpClass(cls):
         """Create class-wide variables."""
         cls.cfg = config.get_config()
+        cls.client = api.Client(cls.cfg, api.json_handler)
+
+    def test_file_decriptors(self):
+        """Test whether file descriptors are closed properly.
+
+        This test targets the following issue:
+        `Pulp #4073 <https://pulp.plan.io/issues/4073>`_
+
+        Do the following:
+        1. Create and sync a repo.
+        2. Install ``lsof``, and issue a command to verify that files in the
+           path ``/var/lib/pulp/`` are closed after the sync.
+        3. Assert that issued command returns `0` opened files.
+        """
+        pkg_mgr = cli.PackageManager(self.cfg)
+        pkg_mgr.raise_if_unsupported(unittest.SkipTest)
+        pkg_mgr.install('lsof')
```

Comment:
@kersommoura I left this as it was committed to pulp_file, but I kind of think it would be better if the test didn't try to do install or uninstall anything - just check if ```lsof``` is available and skip if it isn't installed.  The reason being, if the user already had ```lsof``` installed, wouldn't the next line would remove it from their system?  I don't think we should be doing anything that would make permanent changes to the system configuration of the host machine.  

If we have dependencies like this, it makes sense to document them and have the user (or CI configuration) make sure they're installed if they want that test to be run.

Reply:
If you agree, I don't mind changing it both here and in pulp_file.

Reply:
@dralley, thanks for your working on this. I agree. What about we state in the test case something like...this test requires that `lsof` is available. Otherwise test will be skipped.